packages/aws-client/src/lib/S3MultipartUploads.ts

```diff
@@ -5,13 +5,14 @@ import range from 'lodash/range';
 import {
   AbortMultipartUploadRequest,
   CompleteMultipartUploadRequest,
+  CompleteMultipartUploadOutput as S3CompleteMultipartUploadOutput,
   CreateMultipartUploadRequest,
   UploadPartCopyRequest,
 } from '@aws-sdk/client-s3';
 
 import { s3 } from '../services';
 
-export interface CompleteMultipartUploadOutput extends AWS.S3.CompleteMultipartUploadOutput {
+export interface CompleteMultipartUploadOutput extends S3CompleteMultipartUploadOutput {
```

Comment:
Hmm maybe we can clean this up? It looks like we're importing `CompleteMultipartUploadOutput` as an alias, then exporting `CompleteMultipartUploadOutput` as an extension of that alias?

I guess our interface already shared a name with an AWS interface but the namespace was different. Kinda gross but I see why you did it this way. The grossness was pre-existing.

Maybe we rename our interface? Also maybe this doesn't really matter except that it's a little confusing at first glance.

Reply:
I added the alias because it was the path to least resistance but I think you're right and our interface should really be named something else. We already do something similar here: https://github.com/nasa/cumulus/pull/3618/files#diff-56edb8ddc2b0588b9a8b2f0bb5f33759f187a5bd46e30725d6e79cbe6ed46056L162 although I'm not sure us calling it `SQSMessage` is super clean either. 

Curious your opinion on if we should address both, this one, or neither in this PR.

Reply:
Yeah. What if we took a small step and instead of importing `CompleteMultipartUploadOutput` as an alias, we import it normally and rename _our_ interface `S3CompleteMultipartUploadOutput`?

At least that way we don't have something with exactly the same name as the AWS interface, which could(?) cause confusion.

Reply:
@npauzenga Pushed a small change that refactored that in this file. aws-client unit tests pass locally and waiting on CI now.